var/spack/repos/builtin/packages/ploticus/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Ploticus(MakefilePackage):
+    """Ploticus can produce various types of plots and graphs."""
```

Comment:
The project seems to be abandoned since 2016. According to news there's a "DataDrew" project that took this over this year https://ploticus.sourceforge.net/doc/news.html Is that correct? Why do we need this 2013 package?